css/css-transforms/parsing/scale-parsing-invalid.html

```diff
@@ -14,6 +14,13 @@
 test_invalid_value("scale", "100px");
 
 test_invalid_value("scale", "100 200 300 400");
+
+test_invalid_value("scale", "1%");
```

Comment:
Percentages are valid per this resolution (the PR for which unfortunately hasn't been merged yet): https://github.com/w3c/csswg-drafts/issues/3399

Reply:
OK! I've updated the PR to only be about junk values.